prow/jobs/jinja/presubmits/service_tests.jinja2

```diff
@@ -102,6 +102,27 @@
           value: {{ service }}
         command: ["wrapper.sh", "bash", "-c", "make test-recommended-policy SERVICE=$SERVICE"]
 
+  - name: {{ service }}-codespell
+    decorate: true
+    optional: false
+    always_run: true
+    annotations:
+      karpenter.sh/do-not-evict: "true"
+    spec:
+      serviceAccountName: pre-submit-service-account
+      containers:
+- image: ghcr.io/codespell-project/actions-codespell/stable:v1
+        resources:
+          limits:
+            cpu: 4
```

Comment:
Maybe cpu 4/500Mi is a little too much here?

Reply:
Perhaps? Do you have a suggestion?

Reply:
500Mi should be more than enough. We don't want to waste too many resources on code spelling checks if we don't have to

Reply:
I reduced it to 1/200Mi